app/components/Repos/Card/index.jsx

```diff
@@ -32,24 +32,24 @@ const RepoCard = ({ data }) => (
     {data.map(
       ({ description, id, issues, logo, modifiedDate, name, totalFunded }) => (
         <StyledListItem key={id}>
-          <TitleContainer>
-            <NameLink to={`/repos/detail/${id}`}>{name}</NameLink>
-            <SettingsContainer>
-              <DateWrapper>
-                Last post {moment(modifiedDate).format('M/D/YYYY')}
-              </DateWrapper>
-            </SettingsContainer>
-          </TitleContainer>
+          <ImageContainer>
+            <StyledImageLinkWrapper
+              alt="Repo Image"
+              image={logo}
+              route={`/repos/detail/${id}`}
+              size="8rem"
+              isCircle={false}
+            />
+          </ImageContainer>
           <ContentContainer>
-            <ImageContainer>
-              <StyledImageLinkWrapper
-                alt="Repo Image"
-                image={logo}
-                isSquare
-                route={`/repos/detail/${id}`}
-                size="5rem"
-              />
-            </ImageContainer>
+            <TitleContainer>
+              <NameLink to={`/repos/detail/${id}`}>{name}</NameLink>
+              <SettingsContainer>
```

Comment:
I think we can also get rid of the `Date` on this card. So we can remove the following chunk of code:
```
     <SettingsContainer>
                <DateWrapper>
                  Last post {moment(modifiedDate).format('M/D/YYYY')}
                </DateWrapper>
              </SettingsContainer>
```